extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -351,6 +351,9 @@ export class JupyterSession implements nb.ISession {
 				}
 			}
 
+			// Enable silent mode for perf improvement of reload command.
```

Comment:
does this fix the cmd window popping up, or is that something different?

Reply:
I don't see that locally. But that's something different. If it persists, we need to check with Michael.